sdk/storage/azure-storage-common/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ set(
     inc/azure/storage/common/internal/file_io.hpp
     inc/azure/storage/common/internal/reliable_stream.hpp
     inc/azure/storage/common/internal/shared_key_policy.hpp
+    inc/azure/storage/common/internal/shared_key_policy_lite.hpp
```

Comment:
It's still in `azure-storage-common` package. Can we move this to `azure-data-table` package?

Reply:
There is a debate in the team as to what we should do here, it will be a while till we decide.